public/scripts/controllers/mainController.js

```diff
@@ -0,0 +1,15 @@
+angular.module('scrambleApp').
+  controller('mainController', ['$scope', 'gameStateService',
+             function($scope, gameStateService) {
+    //Sets the current scope of the game into the scope
+
+    $scope.state = gameStateService.getState().state;
```

Comment:
no longer need to expose state to the scope, recommend being careful of adding things to the scope because it makes the scope heavier, making it less performant overall